modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -37,13 +38,7 @@ Do not use this image registry credentials file as the pull secret when you inst
 ====
 endif::restricted[]
 
-ifdef::restricted[]
-[WARNING]
-====
-This process requires that you have write access to a container image registry on the mirror registry and adds the credentials to a registry pull secret.
-====
-
-endif::restricted[]
+// removed this warning since the first part is more of a prereq, and the second part seems to just describe what will happen later in the procedure. Can revert/modify as needed.
```

Comment:
I think it is better to keep this warning as we are working with the registry in the procedure described below.